src/main/resources/static/css/style.css

```diff
@@ -0,0 +1,505 @@
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    width: 100%;
+    font-size: .9em;
+    color: white;
+    }
+
+header {
+    width: 100vw;
+    position: fixed;
+    top: 0;
+    z-index: 10;
+    justify-content: flex-start;
+    display: flex;
+    align-items: center;
+    background-color: #1a4871;
+    color: white;
+    }
+
+#logo_frame {
+    display: inline-block;
+    background-image: url("/img/peaklogo500w.png");
+    background-repeat: no-repeat;
+    background-position: center;
+    }
+
+h1 {
+    font-family: 'Viga', sans-serif;
+    display: inline-block;
+    color: white;
+}
+
+nav{
+    display: inline-block;
+}
+
+nav > div {
+    display: inline-block;
+}
+
+nav > div > a {
+     color: white;
+     text-decoration: none;
+}
+
+i {
+    background: transparent;
+    color: white;
+    font-size: 2.5em;
+}
+
+#nav_icon {
+    background: transparent;
+    color: white;
+    font-size: 2em;
+}
+
+#nav_icon_user {
+    background: transparent;
+    color: white;
+    font-size: 2em;
+}
+
+footer {
+    height: 3vh;
+}
+
+#map {
+    background-color: grey;
+    display: inline-block;
+}
+
+
+#mobile_nav_spacer{
+    display: none;
+}
+
+.updateForm {
+    display: none;
+}
+
+form{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+}
+
+
+@media screen and (orientation:portrait) {
+
+    * {
+
+    }
+
+    body{
+        position: absolute;
+        top: 3vh;
+    }
+
+    head_space {
+        height: 3vh;
+    }
+
+    header {
+        height: 3vh;
+        background:rgba(0, 0, 0, .5);
+    }
+
+    header > nav > div > a {
+         display: none;
+    }
+
+    .top_name{
+            display:none;
+    }
+
+    #teams{
+        display: flex;
+        flex-wrap: wrap;
+        width: 96vw;
+    }
+
+    .team_card{
+        width: 100%;
+        background:rgba(0, 0, 0, 0.5);
+        border-radius: 3px;
+        display: inline-block;
+        margin-top: 5px;
+    }
+
+
+    #mobile_nav_spacer{
+        height: 50px;
+        display:inline-block;
+    }
+
+    #logo_view{
+        float:left;
+        margin-left: 5px;
+        display: flex;
+        align-items: center;
+    }
+
+    #mobile_nav{
+        z-index: 11;
+        position: fixed;
+        top: 3vh;
+        width: 60px;
+        left: 0px;
+        display: flex;
+        justify-content: space-between;
+        margin-top: 5px;
+        height: 50px;
+        align-content: center;
+        align-items: center;
+    }
+
+    #vertical_locations_m{
+        display: none;
+        position: fixed;
+        top: calc(3vh + 55px);
+        z-index: 10;
+        width: 100%;
+        background:rgba(0, 0, 0, 1);
+    }
+
+    #mountain_detail{
+        border-radius: 0px 5px 5px 0px;
+        padding-left: 5px;
+        padding-right: 5px;
+
+    }
+
+    #vertical_locations{
+        display:none;
+    }
+
+    .resort_button_div{
```

Comment:
Web UX 101: Since these are acting like buttons/links, you should add a `cursor: hand` declaration to this style.